src/client.ts

```diff
@@ -16,6 +16,11 @@ import {
 import { Publication } from './messages/Publication';
 import { Toast } from './messages/Toast';
 
+interface ClientConfigOptions {
```

Comment:
TSDoc doesn't require the interface to be commented?

Reply:
Actually, yes and there's a bunch of stuff tslint is complaining about related to properties not being documented.  Guess I've got some work to do...